extensions/transaction/transaction-fragment.yaml

```diff
@@ -0,0 +1,206 @@
+paths:
+  '/collections/{collectionId}/items':
+    post:
+      summary: add a new feature to a collection
+      description: create a new feature in a specific collection
+      operationId: postFeature
+      tags:
+        - Extensions
+      parameters:
+        - $ref: '#/components/parameters/collectionId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/itemCollection'

```

Comment:
Perhaps this could be anyOf:
-item
-itemCollection 

to support single features and collections?